spec/api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe Api::V1::UsersController do
+  describe 'GET /api/users' do
+    it 'returns all users' do
+      10.times do
```

Comment:
I would change it to creating only 3 users each with his own name and email and than check if response contains all of them